style.css

```diff
@@ -0,0 +1,208 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav img {
+  margin: 35px 41px 20px 0;
+}
+
+.descripion {
+  background-image: url(img/backgrounf.png);
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+  margin-top: 92px;
+  padding-bottom: 42px;
+}
+
+.descripion h1,
+h2,
+h3 {
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  font-weight: 700px;
+  text-align: center;
+  line-height: 52px;
+  color: #172b4d;
+  margin: 0;
+}
+
+.descripion h3 {
+  color: #36b37e;
+}

```

Comment:
### [Required Changes]
- Well done but you still have one more thing you omitted. The [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) says "Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels" kindly implement this part. It will be easier for you to style when you start the desktop version of this project